bentoml/_internal/bento/bento.py

```diff
@@ -71,6 +71,7 @@ def create(
 
             model_name, model_version = model_tag.split(":")
             target_path = os.path.join("models", model_name, model_version)
+            bento_fs.makedirs(target_path, recreate=True)
             copy_dir(fs.open_fs(model_info.path), "/", bento_fs, target_path)
```

Comment:
for `copy_dir` here, do we want to `preserve_time=Ture` here? since we will be using that for finding the `latest` model?

Reply:
cc @sauyon